lua/ttt2/libraries/marker_vision.lua

```diff
@@ -294,6 +313,13 @@ if CLIENT then
                 continue
             end
 
+            if isOnScreenCenter then
+                markerVision.focussedMarkers[#markerVision.focussedMarkers + 1] = {
+                    entity = ent,
+                    screenDistanceSquared = screenDistSquared,
+                }
```

Comment:
you seem to add one focussed marker each frame here, am I correct? Do you clear them as well besides when a player presses their use key? Otherwise this seems like a big problem. Also I'm a bit confused with the screenDistSquared thing. Why does it have to be squared?

Reply:
I see, I didn't see that connection while looking at the code, but now I get it